design/20200105-spec-v1alpha2-kubernetes-friendly.md

```diff
@@ -195,19 +195,19 @@ spec:
   - name: instanceName
     required: true
     fieldPaths:
-    - ".metadata.name"
+    - "metadata.name"
   - name: cacheSecret
     required: true
     fieldPaths:
-    - ".spec.containers[0].env[0].value"
+    - "spec.containers[0].env[0].value"
 ```
 
 Components would most frequently be defined by application developers. A
 Component has two responsibilities:
 
-1. Embed a valid OAM workload kind under its `.spec.workload` field.
-2. Publish a set of parameters under its `.spec.parameters` field. Parameters
-   may be set by an ApplicationConfig. Their values overlay onto fields in the
+1. Embed a valid OAM workload kind under its `spec.workload` field.
+2. Publish a set of parameters under its `spec.parameters` field. Parameters may
```

Comment:
Note: these updates are not strictly needed here, but I thought it would be a good idea to mirror the convention we are standardizing on for JSON field paths.